datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxRepairService.java

```diff
@@ -144,18 +142,6 @@ public void waitCloudbreakClusterRepair(Long id, PollingConfig pollingConfig) {
         }
     }
 
-    private boolean hasActiveFlow(SdxCluster sdxCluster) {
-        try {
-            List<FlowLogResponse> flowLogsByResourceNameAndChainId =
-                    flowEndpoint.getFlowLogsByResourceNameAndChainId(sdxCluster.getClusterName(), sdxCluster.getRepairFlowChainId());
-            return flowLogsByResourceNameAndChainId.stream()
-                    .anyMatch(flowLog -> flowLog.getStateStatus().equals(StateStatus.PENDING) || !flowLog.getFinalized());
-        } catch (Exception e) {
-            LOGGER.error("Exception occured during getting flow logs from CB: {}", e.getMessage());
-            return false;
-        }
-    }
-
```

Comment:
If we want to use this generally, we should refactor this because this is working only for repair now